_hackbars/zanoni.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: hackbar
+title: Dry River
+author: Augusto Zanoni
+---
+
+Open Source is about community and help each other to make a better code. Just like the enviroment shoud be treated
+
+---
+
+## <!-- Your MESSAGE on climate change -->
```

Comment:
hey @AugustoZanoni, slipped something on the way 🧐

👉 Add your __MESSAGE__ on climate change

Reply:
Thanks for the heads up, fixed 😄